examples/dbnode/proto_client/README.md

```diff
@@ -0,0 +1,5 @@
+# Protobuf Client Example
+
+1. Setup an M3DB container as described in the `use_as_tsdb.md` file.
+2. Modify `config.yaml` with any changes you've made to default configuration.
+3. Execute `go run main.go -f config.yaml`
```

Comment:
i don't think m3db is using inotify/fsnotify for config, is it? if not, should probably call out that the config needs to be modified prior to running the container, else disappointment/confusion will ensue.